cmake/Package.cmake

```diff
@@ -2,8 +2,8 @@ if(QT_MOC_EXECUTABLE)
   get_filename_component(_qt_bin_dir ${QT_MOC_EXECUTABLE} DIRECTORY)
 endif()
 find_program(LINUXDEPLOY_EXECUTABLE linuxdeploy linuxdeploy-x86_64.AppImage)
-find_program(MACDEPLOYQT_EXECUTABLE macdeployqt HINTS "${_qt_bin_dir}" NO_DEFAULT_PATH)
-find_program(ANDROIDDEPLOYQT_EXECUTABLE androiddeployqt HINTS "${_qt_bin_dir}")
+find_program(MACDEPLOYQT_EXECUTABLE macdeployqt HINTS "${QT_HOST_LIBEXECS}" NO_DEFAULT_PATH)
+find_program(ANDROIDDEPLOYQT_EXECUTABLE androiddeployqt HINTS "$ENV{Qt6_DIR}/../gcc_64/bin")
```

Comment:
Ideally we should adjust lines 1-3 instead. Or if we don't find a solution to find the qt location without the use of env vars we should delete 1-3.